docs/docs/concepts/infrastructure-architecture.mdx

```diff
@@ -0,0 +1,139 @@
+
+Last but not least, the Spawners are abstract interfaces to processes that can start,
+monitor, and stop single-user servers. They are responsible for managing the lifecycle
+of the user's server, including spawning and stopping the server, as well as monitoring
+its status.
+
+![JupyterHub’s Major subsystems](/img/jhub_subsystems.png)
+
+For more information on JupyterHub's architecture, please refer to the official
+[JupyterHub Technical Overview](https://jupyterhub.readthedocs.io/en/stable/reference/technical-overview.html#) documentation.
+
+## JHub Apps Integration
+
+When JHub Apps is launched, it is integrated into JupyterHub as a new service. This addition modifies the Hub's homepage to feature the service's URL, thereby offering users not only access to other services but also control over spawning different frameworks beyond JupyterLab, like Streamlit and Bokeh, while utilizing JupyterHub's authentication and spawner mechanisms.
+
+The newly added Jhub-apps service introduces intermediate steps that tailor request routing based on the selected framework:
+
+- The Hub initiates its own proxy server,
+- Jhub-apps is launched and registered within JupyterHub as an external service,
+- The proxy initially routes all requests to the Hub, while complying with the
+  extensions and modifications made by Jhub-apps,
+- Jhub-apps modifies the Hub's homepage to include its URL (dynamically handled by the
+  proxy above), as well as extending the service
+  selection options for users, who can now choose from a variety of frameworks.
+- The Hub continues to handle logins and server spawning,
+- Jhub-apps adjusts the request handling to direct users to the appropriate single-user
+  server environments based on their selections.
+
+## Technical Architecture Diagram
+
+Below is a diagram illustrating the technical architecture of the JHub Apps service and its interaction with the JupyterHub system:
+
+![JHub Apps Architecture](/img/jhub_service_diagram.png)
+
+Starting from the left of the diagram:
+- Users engage with the JupyterHub interface via their browsers, logging in through
+  their preferred OAuth providers (e.g., GitHub, Google). Authenticated users are
+  redirected to JHub's homepage, where they select and launch the desired service.
+- As an external server managed by JHub, the JHub Apps service maintains full access to
+  JupyterHub’s endpoint API's: authentication, authorization, and access controls. As a result, users
+  still maintain the capability to visit Hub's built-in pages.
+- On the homepage, users can choose the specific framework to deploy as a single-user server. JHub Apps then spawns the selected framework similarly to how JupyterLab is launched. The service redirects users to their individual server instances, where they can interact with the chosen framework.
+
+Once a framework is selected it will determined which Spawner and proxy will be
```

Comment:
This sentence seems to start without context like how do you select a framework, where do you do it, what page we're talking about.

Spawner usually remains the same irrespective of the framework.

Reply:
You are absolutly right, this was a left-over since I moved this all to the `JuoyterHub
subsystems` section, and I forgot to remove it. I will do so now. Thanks for pointing it out.